app/js/components/tabs/tabcomponents/observationComponent.jsx

```diff
@@ -352,7 +352,13 @@ export default class ObsFilter extends React.Component {
           </div>
           <div className="form-group">
               <div className="col-sm-offset-3 col-sm-6">
-                  <button type="submit" className="btn btn-success" >Search</button>
+                  <button
+                    type="submit" 
+                    className="btn btn-success"
+                    onClick={this.handleSubmit && this.handleReset}
```

Comment:
Good work. Just learned how to pass multiple functions to an onclick event here 😀

Reply:
@cnagadya at this point, i think i do not need to convince you any more regarding how valuable code reviews are! 😊

Reply:
Surely, I am now reaping the benefits of code reviews

Reply:
Thank you @cnagadya . 💯